### PR TITLE
deps: tap@15.1.1

### PR DESCRIPTION
**Pull request by isaacs**

### CODE REVIEW

**Comment by wraithgar** [comment]
Tests have not worked since the version of tap that's currently in the cli due to changes in `signal-exit`

**Comment by wraithgar** [comment]
Tests do not pass using anything newer than `tap@15.0.9` due to changes in signal-exit.

File: test/lib/utils/exit-handler.js [hunk 1]
```diff
@@ -38,13 +38,10 @@ t.before(async () => {
   npm.config.set('cache', cacheFolder)
 })
 
-t.test('bootstrap tap before cutting off process ref', (t) => {
```
Comment (wraithgar) [inline test/lib/utils/exit-handler.js:41]: YES!

**Review by wraithgar — APPROVED** [review]

**Comment by fritzy** [comment]
pull script gave me trouble, but it is merged

**Pull request closed without merging**